pkg/server/test/write.go

```diff
@@ -633,6 +633,47 @@ var writeCommandTests = []writeCommandTest{
 			},
 		),
 	},
+	{
+		_name: "ExecuteReturnsErrorIfAuthModelNotFound",
+		// state
+		model: &openfgapb.AuthorizationModel{
+			Id:            ulid.Make().String(),
+			SchemaVersion: typesystem.SchemaVersion1_0,
```

Comment:
We're doing away with 1.0 support, so we should prioritize tests that default to 1.1. Can we switch this specific test case to be for 1.1 please?

Reply:
Done